python/test/t_DistFunc_discrete.py

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env python
+
+from __future__ import print_function
+from openturns import *
+
+TESTPREAMBLE()
+RandomGenerator.SetSeed(0)
+
+try:
+    # Discrete related functions
+    # rDiscrete
+    p = [0.1, 0.2, 0.3, 0.4]
+    print("rDiscrete(", p, ")=%.6g" %
+                      DistFunc.rDiscrete(p))
+    print("rDiscrete(", p, ", 100)=",
```

Comment:
The print of p is formatted differently on the autobuilder and that test fails, could you wrap it in a Point ?
http://autobuilder.openturns.org/openturns/pull/870/squeeze-x86_64-ot-linux/LastTest.log

Reply:
@jschueller Done. I used an Indices instead of a Point as the returned value was an UnsignedInteger.

Reply:
It's "p" that's printed wrong, not the result.

Reply:
@jschueller Ouch! Fixed.